inference-engine/tests/functional/plugin/gpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -9,5 +9,8 @@
 
 std::vector<std::string> disabledTestPatterns() {
     return {
+        // TODO: Issue: 32521
+        R"(.*(EltwiseLayerTest).*secondaryInputType=CONSTANT.*netPRC=FP16.*)",
```

Comment:
be precise here, i am pretty sure that many of the constant FP16 tests are passing